async/retry_test.ts

```diff
@@ -151,3 +151,9 @@ Deno.test("retry() checks backoff function timings", async (t) => {
 
   Math.random = originalMathRandom;
 });
+
+Deno.test("retry() with default options", async () => {
```

Comment:
The default option behavior is tested in `"retry() waits four times by default"` case